hypha/static_src/src/javascript/apply/file-uploads.js

```diff
@@ -1,7 +1,13 @@
-(function ($) {
+jQuery(function ($) {
```

Comment:
Why change this? The original style is how all jQuery code looks on the site.

Happy to see it converted to vanilla JS but that is not important.

Reply:
I changed it when testing jquery-file-upload and dropzone.js. I think was necessary because now we include this file with `Widget.Media`, which (probably) outputs it further up the page, and we need domcontentloaded. If this general approach works then we'll need to clean up this file anyway (and might as well use vanilla JS then).